### PR TITLE
Connect Artwork detail with QR scanner, fix bugs

### PR DESCRIPTION
**Pull request by Sunnysit**

### CODE REVIEW

File: src/tabs/Artwork/components/ArtworkInfo.js [hunk 1]
```diff
@@ -1,92 +1,107 @@
...
-          // console.log(response.data[0].artwork)
-        }
-      })
-      .catch(err => {
-        console.log(err);
-      });
-  }, []);
+    useEffect(() => {
+
+        const receiveSensorId = sensorId
```
Comment (prattrv) [inline src/tabs/Artwork/components/ArtworkInfo.js]: Should this be `receiveSensorId == sensorId`?

Comment (Sunnysit) [reply src/tabs/Artwork/components/ArtworkInfo.js]: Sorry I don't get it. Why is double sign?

Comment (Sunnysit) [reply src/tabs/Artwork/components/ArtworkInfo.js]: The whole line is   `const receiveSensorId = sensorId? sensorId: 'n124';`

Comment (prattrv) [reply src/tabs/Artwork/components/ArtworkInfo.js]: All good! Finally spotted the purpose of this line.

**Pull request merged**